app/code/local/Excellence/Test/controllers/IndexController.php

```diff
@@ -0,0 +1,37 @@
+<?php
+class Excellence_User_IndexController extends Mage_Core_Controller_Front_Action
+{
+    public function deleteAction()
+    {   
+        if( $this->getRequest()->getParam('user_id') > 0 ) {
+         try {
+              $user_model = Mage::getModel('user/user');
+              $address_model = Mage::getModel('user/address');
+              $user_model->setId($this->getRequest()->getParam('user_id'))->delete();
+              $address_model->setId($this->getRequest()->getParam('user_id'))->delete();
+              Mage::getSingleton('adminhtml/session')->addSuccess(Mage::helper('adminhtml')->__('Item was successfully deleted'));
+              $this->_redirect('*/*/index');
+             } 
+         catch (Exception $e) {
+            Mage::getSingleton('adminhtml/session')->addError($e->getMessage());
+            $this->_redirect('*/*/');
+         }
+        }
+      }
+  public function saveAction() {
+    $fields = $this->getRequest()->getParams();
+    $user_model = Mage::getModel("user/user");
+    $address_model = Mage::getModel("user/address");
+    $user_model->setData($fields);
+    $address_model->setData($fields);
+    $user_model->save();
+    $address_model->save();
```

Comment:
address_model needs to have a field called user_id to link user with address. without that its not possible to for which user has what addresss